c2cgeoportal/lib/caching.py

```diff
@@ -120,21 +120,25 @@ def set_common_headers(
     else:  # pragma: nocover
         raise "Invalid cache type"
 
+    settings = request.registry.settings.get("headers", {})
+
     if cache != NO_CACHE:
         max_age = request.registry.settings["default_max_age"]
 
-        settings = request.registry.settings.get("cache_control", {})
-        if service_name in settings and "max_age" in settings[service_name]:
-            max_age = settings[service_name]["max_age"]
+        if service_name in settings and \
+            "cache_control_max_age" in settings[service_name]:
+            max_age = settings[service_name]["cache_control_max_age"]
 
         if max_age != 0:
             response.cache_control.max_age = max_age
         else:
             response.cache_control.no_cache = True
 
-    if add_cors:
+    if add_cors and service_name in settings and \
+        "cors" in settings[service_name]:
```

Comment:
Actually yes, If we want to remove it we should add a `sub_service_name`argument in the function to have a finer granularity.
An than if we do it we should also be able to remove the 
`cache`, `vary` and `content_type` arguments.